mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -2108,7 +2132,7 @@ func flattenTimePartitioning(tp *bigquery.TimePartitioning) []map[string]interfa
 		result["expiration_ms"] = tp.ExpirationMs
 	}
 
-	if tp.RequirePartitionFilter {
+	if tp.RequirePartitionFilter && use_old_rpf {
```

Comment:
When reading the code, it looks like the flag `use_old_rpf` is not needed to check if deprecated require_partition_filter field is used. `tp.RequirePartitionFilter` is enough to check if deprecated require_partition_filter field is used, right?  I am not sure if I miss something.

Reply:
@zli82016 The API returns both the high-level and the deprecated field regardless of which one was set by the user. If I remove the `use_old_rpf` the deprecated field would also be set always.